uMenu/romfs/fr.json

```diff
@@ -0,0 +1,132 @@
+    "user_pass_change_conf": "Voulez-vous changé le mot de passe ?",
+    "user_pass_change_ok": "Le mot de passe a été modifié avec succès.",
+    "user_pass_change_error": "Une erreur s'est produite lors de la tentative de modification du mot de passe",
+    "user_pass_remove_full": "Supprimer le mot de passe",
+    "user_pass_remove_conf": "Souhaitez-vous vraiment supprimer le mot de passe de l'utilisateur ?",
+    "user_pass_remove_ok": "Le mot de passe a été supprimé avec succès.",
+    "user_pass_remove_error": "Une erreur s'est produite lors de la tentative de suppresion du mot de passe",
+    "user_pass_reg_conf": "Souhaitez-vous définir un mot de passe pour cet utilisateur ?",
+    "user_pass_reg_ok": "Le mot de passe a été configuré avec succès.",
+    "user_pass_reg_error": "Une erreur s'est produite lors de la tentative de configuration du mot de passe",
+    "user_logoff_app_suspended": "Il y a un titre ouvert. Souhaitez-vous vraiment le fermer et vous déconnecter ?",
+    "swkbd_webpage_guide": "Entrez l'URL de la page Web",
+    "set_unknown_value": "Unconnu",
+    "set_true_value": "Activé",
+    "set_false_value": "Désactivé",
+    "set_info_text": "Parcourir et/ou modifier les paramètres du système et d'uLaunch",
+    "set_console_nickname": "Pseudonyme de la console",
+    "set_console_timezone": "Emplacement du fuseau horaire de la console",
+    "set_viewer_enabled": "Connexion USB sur la visionneuse d'écran du PC (uViewer) activée",
+    "set_flog_enabled": "Prise de contrôle 'flog' en mode application Homebrew activée",
+    "set_wifi_none": "aucun (pas de connexion WiFi)",
+    "set_wifi_name": "Réseau WiFi connecté",
+    "set_console_lang": "Langue de la Console",
+    "set_console_info_upload": "Téléchargement des informations de la console activé",
+    "set_auto_titles_dl": "Téléchargement automatique des titres activé",
+    "set_auto_update": "Mise à jour automatique de la console activée",
+    "set_wireless_lan": "LAN sans fil activé",
+    "set_bluetooth": "Bluetooth activé",
+    "set_usb_30": "USB 3.0 activé",
+    "set_nfc": "NFC activé (amiibo)",
+    "set_serial_no": "Numéro de serie de la Console",
+    "set_mac_addr": "Adresse MAC",
+    "swkbd_console_nick_guide": "Entrer le nouveau nom de la console",
+    "set_enable_conf": "Voulez-vous l'activé ?",
+    "set_disable_conf": "Voulez-vous le désactivé ?",
+    "set_changed_reboot": "Succès. Un redémarrage est nécessaire (l'option ne sera pas utilisée d'ici là)",
+    "set_viewer_info": "Vous devez l'activer si vous souhaitez utiliser la visionneuse d'écran du PC (uViewer). Ce n'est pas nécessaire sinon.",
+    "set_flog_info": "Celui-ci doit être activé pour pouvoir lancer des homebrews directement en tant qu'applications. Notez que cela peut impliquer un RISQUE D'INTERDICTION.",
+    "startup_welcome_info": "Bienvenue ! Veuillez sélectionner un compte à utiliser.",
+    "startup_login_error": "Mot de passe incorrect. Veuillez réessayer svp.",
```

Comment:
-svp

Reply:
no need -

Reply:
Sorry, I meant to remove the "svp". I think it is not needed.

Reply:
no, svp is needed after that...

Reply:
Well, even if original says "Please", you are not forced to find a replacement for it. Usually, softwares will not say "svp" :D.

Reply:
lol